sapi_manifests/madtom_coal/template

```diff
@@ -57,42 +56,13 @@
         { "processType": "agent", "port": 9080 }
       ]
     },
-    {
-      "hostType": "jobsupervisor",
-      "processes": [
-        { "processType": "supervisor", "port": 80 }
-      ]
-    },
     {
       "hostType": "nameservice",
       "processes": [
         { "processType": "binder", "port": 53 },
         { "processType": "zookeeper", "port": 2181 }
       ]
     },
-    {
-      "hostType": "medusa",
-      "processes": [
-        { "processType": "medusa", "port": 8381 }
-      ]
-    },
-    {
-      "hostType": "jobpuller",
-      "processes": [
-        { "processType": "wrasse", "port": 80 }
-      ]
-    },
-    {
-      "hostType": "ops",
```

Comment:
While I would like to drop mola (the "ops" service, https://jira.joyent.us/browse/MANTA-5099), it technically still exists. I'm fine either keeping or dropping it here.

Reply:
The 'ops' zone is here to avoid the kind of error that db36de5 has fixed (I realized about this too late - we could have just added some empty blocks for the services that have no health checks). The ops section doesn't have any processType defined and no longer serves any function, now that we have the fix to skip the error. So I'll still take it out.